src/security/passwords.py

```
@@ -1,10 +1,59 @@
+import os
+from datetime import datetime, timedelta
+
+from jose import JWTError, jwt
 from passlib.context import CryptContext
+from sqlalchemy import select
+
+from security.token_manager import JWTAuthManager
+
+pwd_context = CryptContext(schemes=["bcrypt"], bcrypt__rounds=14, deprecated="auto")
+
+
+SECRET_KEY = "mysecretkey"
+ALGORITHM = "HS256"
+ACCESS_TOKEN_EXPIRE_MINUTES = 30
+
+
+def create_access_token(data: dict, expires_delta: timedelta = None):
+    to_encode = data.copy()
+    if expires_delta:
+        expire = datetime.utcnow() + expires_delta
+    else:
+        expire = datetime.utcnow() + timedelta(minutes=15)
+    to_encode.update({"exp": expire})
+    encoded_jwt = jwt.encode(to_encode, SECRET_KEY, algorithm=ALGORITHM)
+    return encoded_jwt
+
+
+def decode_token(token: str):
+    try:
+        payload = jwt.decode(token, SECRET_KEY, algorithms=[ALGORITHM])
+        return payload
+    except JWTError:
+        return None
+
+
+async def remove_token(db, user) -> str:
+    from database import PasswordResetTokenModel
+
+    user_token_to_remove = await db.execute(
+        select(PasswordResetTokenModel).where(
+            PasswordResetTokenModel.user_id == user.id
+        )
+    )
+    tokens_to_remove = user_token_to_remove.scalars().all()
+    for token in tokens_to_remove:
+        await db.delete(token)
+    await db.commit()
+    return "Done"
+
 
-pwd_context = CryptContext(
-    schemes=["bcrypt"],
-    bcrypt__rounds=14,
-    deprecated="auto"
-)
+def get_jwt_manager() -> JWTAuthManager:
+    secret_access = os.getenv("SECRET_KEY_ACCESS") or "SECRET_KEY_ACCESS"
+    secret_refresh = os.getenv("SECRET_KEY_REFRESH") or "SECRET_KEY_REFRESH"
+    algorithm = os.getenv("JWT_SIGNING_ALGORITHM") or "HS256"
+    return JWTAuthManager(secret_access, secret_refresh, algorithm)
```

`get_jwt_manager` falls back to placeholder strings when environment variables are not set. That's dangerous during real runs because it silently uses weak defaults. Prefer raising an error or using application settings (injected via DI) when secrets are missing. If tests rely on defaults, make that explicit in docs but avoid shipping defaults in production code.